gobblin-temporal/src/main/java/org/apache/gobblin/temporal/workflows/metrics/TemporalEventTimer.java

```diff
@@ -40,61 +41,73 @@
 public class TemporalEventTimer implements EventTimer {
   private final SubmitGTEActivity trackingEventActivity;
   private final GobblinEventBuilder eventBuilder;
-  private final TrackingEventMetadata trackingEventMetadata;
+  private final EventSubmitterContext _eventSubmitterContext;
```

Comment:
nit: none of the other members have `_` prefix... why this one?

Reply:
good catch. This was an artifact of the intellij refactor